src/utils/axios.js

```diff
@@ -0,0 +1,14 @@
+import * as realAxios from 'axios';
+import { BASE_URL, USER_SESSION } from './constants';
+
+const { accessToken } = JSON.parse(
+  localStorage.getItem(USER_SESSION)
+).authResult;
+const axios = realAxios.create({
+  baseURL: BASE_URL,
+  headers: {
+    Authorization: `Bearer ${accessToken}`,
+  },
+});
+
+export default axios;
```

Comment:
I just thought of something else. We could use [axios interceptors](https://github.com/axios/axios#interceptors) to attach the Authorization header. We would intercept every request and add a Authorization header if the user is logged in. The benefit of this is that we won't need to create a new instance of axios of every request and we would be able to use axios without having to wrap it in a utility.

Reply:
I think we can add it in `src/App.jsx`. `authContext.user` would give us access to the user object.

```jsx
axios.interceptors.request.use(config => {
    const accessToken = // get access token
    config.headers.Authorization =  token;

    return config;
});
```